src/main/java/frc/team2412/robot/commands/arm/ManualArmOverrideCommand.java

```diff
@@ -0,0 +1,37 @@
+package frc.team2412.robot.commands.arm;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.team2412.robot.subsystems.ArmSubsystem;
+import java.util.function.DoubleSupplier;
+
+public class ManualArmOverrideCommand extends CommandBase {
+
+	private ArmSubsystem armSubsystem;
+	private DoubleSupplier armJoystickInput;
+	private DoubleSupplier wristJoystickInput;
+
+	public ManualArmOverrideCommand(
+			ArmSubsystem armSubsystem,
+			DoubleSupplier armJoystickInput,
+			DoubleSupplier wristJoystickInput) {
+		this.armSubsystem = armSubsystem;
+		this.armJoystickInput = armJoystickInput;
+		this.wristJoystickInput = wristJoystickInput;
+	}
+
+	@Override
+	public void initialize() {
+		armSubsystem.setManualOverride(true);
```

Comment:
We should probably set manual override to false either when this command ends or when the other commands run if we want to be able to use the presets after using manual override.